examples/CSVImporter/CSVImporter.go

```diff
@@ -199,11 +215,10 @@ func ProcessCSVRecord(jc jcapi.JCAPI, userList []jcapi.JCUser, csvRecord []strin
 
 			// Create or modify the tag in JumpCloud
 			tempTag.Id, err = jc.AddUpdateTag(opCode, tempTag)
-
 			if err != nil {
-				fmt.Printf("Could not process tag '%s', err='%s'", tempTag.ToString(), err)
+				err = fmt.Errorf("Could not POST tag '%s', err='%s'", tempTag.ToString(), err)
 			} else {
-				fmt.Printf("Processed tag ID '%s'\n", tempTag.Id)
+				fmt.Printf("\tCreated tag '%s' (ID '%s')\n", tempTag.Name, tempTag.Id)
```

Comment:
They are on a new-line, and it looks like:

Line #1:
	User 'mlansky' (ID '553028cd7bb78c4a1103f559') updated from input file
	Tag 'Win81Dev - Meyer Lansky (mlansky)' already exists, not modifying it.
Line #2:
	User 'vito' (ID '553028cf00678e380b03f2f6') updated from input file
	Tag 'Win81Dev - Vito Corleone (vito)' already exists, not modifying it.